wal/repair.go

```diff
@@ -86,10 +86,12 @@ func Repair(lg *zap.Logger, dirpath string) bool {
 				return false
 			}
 
+			start := time.Now()
 			if err = fileutil.Fsync(f.File); err != nil {
```

Comment:
Would it make sense to calculate metrics inside `fileutil.Fsync` so that a) consumer won't forget to add metrics in new places where `Fsync` is called and 2) capture the latency even for the failed calls?

Reply:
`fileutil.Fsync` is used in other packages, so we don't want to break the interface and introduce a new prometheus metrics for a simple package. We may consider wrapper in the next release, so we need not do this.